.travis.yml

```diff
@@ -21,8 +21,8 @@ script:
   - cd ..
 
   - cd frontend
-  - ng lint
-  - ng test --single-run --code-coverage --browsers HeadlessChrome
+  - $(npm bin)/ng lint
```

Comment:
if we are going to use local `ng` script, You can remove global `@angular/cli` installation.

Reply:
https://github.com/CodeForPoznan/volontulo/pull/1040/commits/5c12fb7c3bcabac0c19c5d3bdfbed9341225f0ee#diff-354f30a63fb0907d4ad57269548329e3L11

Removed